File: code_samples/python/v1.0.0/set_colors.py
from tatrapayplus import TatrapayPlusClient
from tatrapayplus.models import *

client = TatrapayPlusClient(
    "https://api.tatrabanka.sk/tatrapayplus/sandbox",
    "your-client-id",
    "your-client-secret"
)

appearance_data = AppearanceRequest(
    theme=AppearanceRequestTheme.SYSTEM,
    surface_accent=ColorAttribute(color_dark_mode="#ff0000", color_light_mode="#ff0000"),
    tint_accent=ColorAttribute(color_dark_mode="#00ff00 ", color_light_mode="#00ff00"),
    tint_on_accent=ColorAttribute(color_dark_mode="#0000ff", color_light_mode="#0000ff"),
)
response = client.set_appearance(appearance_data)

File: code_samples/php/v1.0.0/set_colors.php
<?php
use Tatrapayplus\TatrapayplusApiClient\Api\TatraPayPlusAPIApi;
use Tatrapayplus\TatrapayplusApiClient\Model\AppearanceRequest;
use Tatrapayplus\TatrapayplusApiClient\Model\ColorAttribute;

$tatrapayplus_api = new TatraPayPlusAPIApi(
    "your-client-id",
    "your-client-secret",
);
$appearance_request = new AppearanceRequest([
    "theme" => AppearanceRequest::THEME_SYSTEM,
    "tint_on_accent" => new ColorAttribute([
        "color_dark_mode" => "#fff",
        "color_light_mode" => "#fff",
    ]),
    "tint_accent" => new ColorAttribute([
        "color_dark_mode" => "#fff",
        "color_light_mode" => "#fff",
    ]),
    "surface_accent" => new ColorAttribute([
        "color_dark_mode" => "#fff",
        "color_light_mode" => "#fff",
    ]),
]);

$response = $tatrapayplus_api->setAppearance($appearance_request);